components/messenger.rst

```diff
@@ -0,0 +1,189 @@
+.. index::
+   single: Messenger
+   single: Components; Messenger
+
+The Messenger Component
+=======================
+
+    The Messenger component helps applications send and receive messages to/from other applications or via message queues.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/messenger
+
+Alternatively, you can clone the `<https://github.com/symfony/messenger>`_ repository.
+
+.. include:: /components/require_autoload.rst.inc
+
+Concepts
+--------
+
+.. image:: /_images/components/messenger/overview.png
+
+**Sender**:
+   Responsible for serializing and sending the message to _something_. This
+   something can be a message broker or a third party API for example.
```

Comment:
`a messager handler, broker or a third-party API`.

Reasons:

- The current phrasing kinda implies the component is very queue oriented. It's not wrong but it's not limited to this use case
- Using it with a message handler is a case common enough to be mentioned IMO
- I'm not sure all the people are familiar with queues and its related vocabulary so putting `message handler` which is likely to be more commonly known will make it more accessible